Library/Homebrew/requirements.rb

```diff
@@ -114,10 +114,15 @@ def message
   end
 end
 
+class CVSRequirement < Requirement
+  fatal true
+  default_formula "cvs"
+  satisfy { which "cvs" }
```

Comment:
Do we still want the `MacOS.version >= :mavericks || !MacOS::Xcode.provides_cvs?` check on Mac? If not, is `MacOS::Xcode.provides_cvs?` still used anywhere else?

Reply:
It looks like `Xcode.provides_cvs?` is not used anywhere else, and we could get rid of it.
If we wanted to keep `MacOS.version >= :mavericks || !MacOS::Xcode.provides_cvs?`, then that check could be moved to `extend/os/mac/requirements.rb`.

Reply:
I've modified the PR to remove `MacOS::Xcode.provides_cvs?`.